grails-app/views/inventory/listExpiredStock.gsp

```diff
@@ -47,10 +47,15 @@
 							</a>
 						</div>
 
-						<g:link params="[format:'csv',category:params.category]" controller="${controllerName}" action="${actionName}" class="button">
+						<g:link params="[format:'csv',category: params.category]" controller="inventory" action="listExpiredStock" class="button">
```

Comment:
there are a lot of params, maybe it would be more readable if we split those into new lines?